src/layerstack.stanza

```diff
@@ -536,3 +547,44 @@ public defn create-pcb-stackup (stack:LayerStack) :
       make-layer-statement(l)
 
   custom-stackup
+
+doc:\<DOC>
+  Construct a LayerStack with a tuple of outer layers. 
+
+  Example:
+  ```
+    val copper-35um = Copper(0.035)
+    val copper-17_5um = Copper(0.0175)
+    val jlcpcb-jlc2313 = make-layer-stack("JLCPCB 4-layer 1.6mm", outer-layers,
+      soldermask = soldermask) where :
+      val soldermask = Soldermask(0.019, SoldermaskMaterial)
+      val prepreg-2313 = FR4(0.1, FR4-Material-2313)
+      val core-2313 = FR4(1.265, FR4-Material-Core)
+      val outer-layers = [
+        [copper-35um prepreg-2313] 
+        [copper-17_5um core-2313]
+      ]
+  ```
+  The LayerStack created has these layers:
+     [ soldermask
+       copper-35um prepreg-2313
+       copper-17_5um core-2313
+       copper-17_5um core-2313
+       copper-35um prepreg-2313
```

Comment:
This is incorrect - the resultant layer structure should be:

```
[
  soldermask
  copper-35um 
  prepreg-2313
  copper-17_5um 
  core-2313
  copper-17_5um 
  prepreg-2313
  copper-35um 
  soldermask
]
```

Notice the singular `core-2313` in the center.

Reply:
I noticed that and already corrected it in the last commit.